pandas/tslib.pyx

```diff
@@ -2722,6 +2722,12 @@ class Timedelta(_Timedelta):
     __pos__ = _op_unary_method(lambda x: x, '__pos__')
     __abs__ = _op_unary_method(lambda x: abs(x), '__abs__')
 
+
+# Resolution is in nanoseconds
+Timedelta.min = Timedelta(np.iinfo(np.int64).min+1, 'ns')
+Timedelta.max = Timedelta(np.iinfo(np.int64).max, 'ns')
+Timedelta.resolution = Timedelta(1, 'ns')
```

Comment:
I figured there is something I missing about that property as it breaks.

However, without that change

```python
import pandas as pd
import numpy as np

rng = pd.timedelta_range('1 day 10:11:12', freq='h', periods=500)
s = pd.Series(np.arange(len(rng)), index=rng)
s['5 day':'6 day']
```

gives

```
5 days 00:11:12     86
5 days 01:11:12     87
5 days 02:11:12     88
5 days 03:11:12     89
5 days 04:11:12     90
5 days 05:11:12     91
5 days 06:11:12     92
5 days 07:11:12     93
5 days 08:11:12     94
5 days 09:11:12     95
5 days 10:11:12     96
5 days 11:11:12     97
5 days 12:11:12     98
5 days 13:11:12     99
5 days 14:11:12    100
5 days 15:11:12    101
5 days 16:11:12    102
5 days 17:11:12    103
5 days 18:11:12    104
5 days 19:11:12    105
5 days 20:11:12    106
5 days 21:11:12    107
5 days 22:11:12    108
5 days 23:11:12    109
6 days 00:11:12    110
6 days 01:11:12    111
6 days 02:11:12    112
6 days 03:11:12    113
6 days 04:11:12    114
6 days 05:11:12    115
6 days 06:11:12    116
6 days 07:11:12    117
6 days 08:11:12    118
6 days 09:11:12    119
6 days 10:11:12    120
6 days 11:11:12    121
6 days 12:11:12    122
6 days 13:11:12    123
6 days 14:11:12    124
6 days 15:11:12    125
6 days 16:11:12    126
6 days 17:11:12    127
6 days 18:11:12    128
6 days 19:11:12    129
6 days 20:11:12    130
6 days 21:11:12    131
6 days 22:11:12    132
6 days 23:11:12    133
```
The 6th day is included, yet with the `.resolution` change it is not. The later is what I would expect, but given how explicit the tests were I did not really think they were wrong. That is the source of my confusion.